dids/src/test/kotlin/web5/sdk/dids/methods/dht/DidDhtTest.kt

```diff
@@ -252,8 +290,12 @@ class DidDhtTest {
 
       val otherKey = manager.generatePrivateKey(JWSAlgorithm.ES256K, Curve.SECP256K1)
       val publicKeyJwk = manager.getPublicKey(otherKey).toPublicJWK()
-      val verificationMethodsToAdd: Iterable<Pair<JWK, Array<PublicKeyPurpose>>> = listOf(
-        Pair(publicKeyJwk, arrayOf(PublicKeyPurpose.AUTHENTICATION, PublicKeyPurpose.ASSERTION_METHOD))
+      val verificationMethodsToAdd = listOf(
+        JsonWebKey2020VerificationMethod(
```

Comment:
noting that we should change this to just `JsonWebKey` https://github.com/TBD54566975/did-dht-method/issues/60 https://github.com/TBD54566975/sdk-development/issues/73